news/2018-09-25-project-loved-one-year-week-of-september-23rd.md

```diff
@@ -0,0 +1,235 @@
+- [\_underjoy](https://osu.ppy.sh/users/2235750), [-\[ ix Ishida xi \]-](https://osu.ppy.sh/users/242910), [-Kazu-](https://osu.ppy.sh/users/920861), [aitor98](https://osu.ppy.sh/users/3154852), [Ascendance](https://osu.ppy.sh/users/2931883), [Backfire](https://osu.ppy.sh/users/263110), [Deif](https://osu.ppy.sh/users/318565), [DigitalHypno](https://osu.ppy.sh/users/4384207), [Gezoda](https://osu.ppy.sh/users/481582), [Halogen-](https://osu.ppy.sh/users/169992), [iceOC](https://osu.ppy.sh/users/5482401), [JBHyperion](https://osu.ppy.sh/users/4879508), [Kaifin](https://osu.ppy.sh/users/2596942), [Kamikaze](https://osu.ppy.sh/users/2124783), [Nwolf](https://osu.ppy.sh/users/1910766), [nyanmi-1828](https://osu.ppy.sh/users/6866480), [Pope Gadget](https://osu.ppy.sh/users/2288341), [Sartan](https://osu.ppy.sh/users/4100941), [Shoegazer](https://osu.ppy.sh/users/2520707), [Spectator](https://osu.ppy.sh/users/702598), [stupud man](https://osu.ppy.sh/users/2141612), [tasuke912](https://osu.ppy.sh/users/2774767), [Tenshichan](https://osu.ppy.sh/users/1101600), [TheToaphster](https://osu.ppy.sh/users/7616811), [Toy](https://osu.ppy.sh/users/2757689) and [Zak](https://osu.ppy.sh/users/1375955), for picking maps every round and writing the descriptions on each listing
+- [Gabe](https://osu.ppy.sh/users/654108), [Gero](https://osu.ppy.sh/users/1467715), [Kurai](https://osu.ppy.sh/users/77089), [Mao](https://osu.ppy.sh/users/2204515) and [Noffy](https://osu.ppy.sh/users/1541323), for checking the content and metadata of Loved candidates to make sure that they meet community guidelines
+- [BeasttrollMC](https://osu.ppy.sh/users/3171691) and [\_verto\_](https://osu.ppy.sh/users/2015300), for making awesome videos to promote Project Loved and Loved maps
+- Me ([clayton](https://osu.ppy.sh/users/3666350)), for writing all of these posts :^)
+- [Joehu](https://osu.ppy.sh/users/8549835), [MegaApple\_Pi](https://osu.ppy.sh/users/2148208), [Repflez](https://osu.ppy.sh/users/201392), [Shiro](https://osu.ppy.sh/users/113005) and [TPGPL](https://osu.ppy.sh/users/3944705), for regularly reveiwing the news posts on GitHub before they go live
+- All of you, for casting your votes in the Project Loved forum!
+
+All of these people contribute or have contributed in the past to Project Loved, so make sure to thank them if you enjoy what we do!
+
+This round, we've got some new people helping out. Along with our new osu!taiko captains mentioned last round ([nyanmi-1828](https://osu.ppy.sh/users/6866480) and [-\[ ix Ishida xi \]-](https://osu.ppy.sh/users/242910)), we also have a new osu!mania captain: [stupud man]() (nice)!
+
+The 90% threshold for osu!standard that we tried last round seemed to work out pretty well, so we'll be keeping it around. Also, [\_verto\_](https://osu.ppy.sh/users/2015300) made a video showcasing all of the osu!standard maps again this week, so check it out!
```

Comment:
These two sentences seem a little awkward being bundled together, since they are discussing totally different things. Maybe the first sentence can go at the end of the next paragraph where you mention the beatmap selection, and verto's video can go in the paragraph above where you talk about the people involved?

Reply:
lmk how it is after 0f0857759a79a33e3f42f69750917e6c5a788707